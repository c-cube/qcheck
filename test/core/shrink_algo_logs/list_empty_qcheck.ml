open QCheck

let lists_are_empty_issue_64 =
  Test.make ~name:"lists are empty"
    (list small_int) (Log.shrinks Print.(list int) @@ fun xs -> xs = [])

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand lists_are_empty_issue_64 with Test.Test_fail _ -> ()
