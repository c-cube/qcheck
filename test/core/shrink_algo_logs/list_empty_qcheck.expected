fails [9; 3; 94; 3]
fails [94; 3]
fails [3]
holds []
fails [0]
holds []
